src/backend/devolio-api/src/shared/typeorm/typeorm.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable, Inject } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { TypeOrmOptionsFactory, TypeOrmModuleOptions } from '@nestjs/typeorm';
+
+@Injectable()
+export class TypeOrmConfigService implements TypeOrmOptionsFactory {
+  @Inject(ConfigService)
+  private readonly config: ConfigService;
+
+  public createTypeOrmOptions(): TypeOrmModuleOptions {
+    return {
+      type: 'postgres',
+      host: this.config.get<string>('DATABASE_HOST'),
+      port: this.config.get<number>('DATABASE_PORT'),
+      database: this.config.get<string>('DATABASE_NAME'),
+      username: this.config.get<string>('DATABASE_USER'),
+      password: this.config.get<string>('DATABASE_PASSWORD'),
+      entities: ['dist/**/*.entity.{ts,js}'],
+      migrations: ['dist/migrations/*.{ts,js}'],
+      migrationsTableName: 'typeorm_migrations',
+      logger: 'file',
+      synchronize: true, // never use TRUE in production!
```

Comment:
If we're not supposed to use synchronize in prod, maybe retrieve it from config?